apis/policies/v1/clusteradmissionpolicy_webhook.go

```diff
@@ -46,7 +46,7 @@ func (r *ClusterAdmissionPolicy) SetupWebhookWithManager(mgr ctrl.Manager) error
 	return nil
 }
 
-//+kubebuilder:webhook:path=/mutate-policies-kubewarden-io-v1alpha2-clusteradmissionpolicy,mutating=true,failurePolicy=fail,sideEffects=None,groups=policies.kubewarden.io,resources=clusteradmissionpolicies,verbs=create;update,versions=v1alpha2,name=mclusteradmissionpolicy.kb.io,admissionReviewVersions={v1,v1beta1}
+//+kubebuilder:webhook:path=/mutate-policies-kubewarden-io-v1-clusteradmissionpolicy,mutating=true,failurePolicy=fail,sideEffects=None,groups=policies.kubewarden.io,resources=clusteradmissionpolicies,verbs=create;update,versions=v1;v1alpha2,name=mclusteradmissionpolicy.kb.io,admissionReviewVersions={v1,v1beta1}
```

Comment:
Same here, `v1alpha2` is still mentioned

Reply:
fixed in https://github.com/kubewarden/kubewarden-controller/pull/247/commits/84a0ba1a5d5a925b1cafd7e40e4ce9892dc1a62e

Reply:
commit reverted https://github.com/kubewarden/kubewarden-controller/pull/247/commits/348e840158cb069e358534e89051c10e56b10ec1